docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
```

Comment:
I think is not a good idea depending on "moving pictures", so I guess that CentOS 6.6 should be used here.